src/common/components/Input/index.tsx

```diff
@@ -1,3 +1,126 @@
-import Input from './Input'
+import React, { ReactNode, ReactSVGElement } from 'react'
+import cn from 'classnames'
+import Icon from '../Icon'
+import CheckIcon from '../../assets/icons/ico-check.svg'
+import TimesIcon from '../../assets/icons/ico-times.svg'
+
+export type InputProps = {
```

Comment:
@muhadovic add `T` as a prefix

Reply:
could you rename `input.stories.tsx` to `Input.stories.tsx` for consistency?
![image](https://user-images.githubusercontent.com/55074937/121098354-abcfbb00-c7c3-11eb-8c46-696c31f0b8b5.png)